Openstack_antelope_installation/maas_and_juju_install_and_configure/install_maas_juju.yaml

```diff
@@ -0,0 +1,29 @@
+- name: Install MAAS, JUJU
+  hosts: localhost
+  vars_files:
+    - config.yaml
+  tasks:
+    # assuming
+
+    # - name: Set proxy
+    #   shell: |
+    #     export no_proxy={{no_proxy}}
+    #     export http_proxy={{http_proxy}}
+    #     export https_proxy={{http_proxy}}
+
+    - name: Install MAAS
+      shell: |
+        sudo -E apt-add-repository ppa:maas/3.3
```

Comment:
Please use "become" instead of using sudo in the shell command. Also use existing ansible modules over command or shell, wherever possible (https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_repository_module.html).